test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -811,8 +811,9 @@ private static String removeMacExpectedVars(String vars) {
      */
     private static String removeAixExpectedVars(String vars) {
         String cleanedVars = vars.replace("AIXTHREAD_GUARDPAGES=0,", "");
-        // OpenJ9 adds MALLOCOPTIONS
+        // OpenJ9 adds MALLOCOPTIONS and LDR_CNTRL
         cleanedVars = cleanedVars.replace("MALLOCOPTIONS=multiheap,considersize,", "");
+        cleanedVars = cleanedVars.replace("LDR_CNTRL=TEXTPSIZE=64K@DATAPSIZE=64K@STACKPSIZE=64K@SHMPSIZE=64K,", "");
```

Comment:
The method comment should be updated to list all three expected variables.
Likewise, the comment here should name and the code should remove the variables in alphabetic order (the sentence in the comment should end with a period).